core/store/src/migrations.rs

```diff
@@ -269,12 +306,12 @@ pub fn migrate_38_to_39(store: &Store) -> anyhow::Result<()> {
     }
 
     #[derive(BorshDeserialize)]
-    struct LegacyEpochSummary {
+    struct LegacyEpochSummaryV38 {
```

Comment:
Random observation - those structs are defined inside of the migrate methods while the other ones are on the outside. mini nit - make it consistent